.github/dependabot.yml

```diff
@@ -47,4 +52,5 @@ updates:
   commit-message:
     prefix: ":seedling:"
   labels:
+    - "area/dependency"
```

Comment:
As this label already exists in CAPV, I guess there is no reason why we can't add it